_posts/events/2024-10-03-METACOL.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+categories: [event]
+event_name: MetaCol – Collocations in the Light of Metaphors and Metonymies
+event_url: https://forms.gle/xbgdSBKfTDc9FRjr9
+event_location: DUBROVNIK, CROATIA
+event_submission_date: 2024-01-31
+---
+
+We would like to invite you to participate in the International Linguistic Conference MetaCol – Collocations in the Light of Metaphors and Metonymies, which will be held on 3 and 4 October 2024 and will take place at the Inter-University Center in Dubrovnik, Croatia. 
+The aim of the conference is to provide various insights into the principles underlying the creation of collocations and the linguistic description of its semantic and pragmatic aspects, while bringing the metaphorical and metonymical dimension of collocations to the foreground and discussing also their possible implications for applied linguistics. Proposals can include, but are not limited to studies dealing with metaphorical and metonymical collocations from the perspectives of: 
+•	Semantics (e.g., degree of lexicalization, motivation/transparency etc.) 
```

Comment:
For bulleted lists, use a "-, +, or * character" as per https://github.github.com/gfm/#list-items